### ISSUE TITLE
WordPress database error Multiple primary key defined for query ALTER TABLE wp_bp_activity

### ISSUE THREAD

**Issue by ianmjones** [buddyboss/buddyboss-platform#2655]
1. Clean install of BuddyBoss Platform 1.7.2.2 on clean install of WP5.8 with PHP7.4 & MariaDB 10.3
2. Enable media uploads, including Photos, Documents and Videos
3. Produces the following debug.log output:

```
[21-Jul-2021 15:40:38 UTC] WordPress database error Multiple primary key defined for query ALTER TABLE wp_bp_activity CHANGE COLUMN `id` id bigint(20) NOT NULL AUTO_INCREMENT PRIMARY KEY made by do_action('admin_init'), WP_Hook->do_action, WP_Hook->apply_filters, bp_admin_init, do_action('bp_admin_init'), WP_Hook->do_action, WP_Hook->apply_filters, bp_core_admin_components_activation_handler, bp_core_install, bp_core_install_activity_streams, dbDelta
[21-Jul-2021 15:40:38 UTC] WordPress database error Multiple primary key defined for query ALTER TABLE wp_bp_activity_meta CHANGE COLUMN `id` id bigint(20) NOT NULL AUTO_INCREMENT PRIMARY KEY made by do_action('admin_init'), WP_Hook->do_action, WP_Hook->apply_filters, bp_admin_init, do_action('bp_admin_init'), WP_Hook->do_action, WP_Hook->apply_filters, bp_core_admin_components_activation_handler, bp_core_install, bp_core_install_activity_streams, dbDelta
[21-Jul-2021 15:40:38 UTC] WordPress database error Multiple primary key defined for query ALTER TABLE wp_bp_notifications CHANGE COLUMN `id` id bigint(20) NOT NULL AUTO_INCREMENT PRIMARY KEY made by do_action('admin_init'), WP_Hook->do_action, WP_Hook->apply_filters, bp_admin_init, do_action('bp_admin_init'), WP_Hook->do_action, WP_Hook->apply_filters, bp_core_admin_components_activation_handler, bp_core_install, bp_core_install_notifications, dbDelta
[21-Jul-2021 15:40:38 UTC] WordPress database error Multiple primary key defined for query ALTER TABLE wp_bp_notifications_meta CHANGE COLUMN `id` id bigint(20) NOT NULL AUTO_INCREMENT PRIMARY KEY made by do_action('admin_init'), WP_Hook->do_action, WP_Hook->apply_filters, bp_admin_init, do_action('bp_admin_init'), WP_Hook->do_action, WP_Hook->apply_filters, bp_core_admin_components_activation_handler, bp_core_install, bp_core_install_notifications, dbDelta
[21-Jul-2021 15:40:38 UTC] WordPress database error Multiple primary key defined for query ALTER TABLE wp_bp_follow CHANGE COLUMN `id` id bigint(20) NOT NULL AUTO_INCREMENT PRIMARY KEY made by do_action('admin_init'), WP_Hook->do_action, WP_Hook->apply_filters, bp_admin_init, do_action('bp_admin_init'), WP_Hook->do_action, WP_Hook->apply_filters, bp_core_admin_components_activation_handler, bp_core_install, bp_core_install_follow, dbDelta
[21-Jul-2021 15:40:38 UTC] WordPress database error Multiple primary key defined for query ALTER TABLE wp_bp_xprofile_groups CHANGE COLUMN `id` id bigint(20) unsigned NOT NULL AUTO_INCREMENT PRIMARY KEY made by do_action('admin_init'), WP_Hook->do_action, WP_Hook->apply_filters, bp_admin_init, do_action('bp_admin_init'), WP_Hook->do_action, WP_Hook->apply_filters, bp_core_admin_components_activation_handler, bp_core_install, bp_core_install_extended_profiles, dbDelta
[21-Jul-2021 15:40:38 UTC] WordPress database error Multiple primary key defined for query ALTER TABLE wp_bp_xprofile_fields CHANGE COLUMN `id` id bigint(20) unsigned NOT NULL AUTO_INCREMENT PRIMARY KEY made by do_action('admin_init'), WP_Hook->do_action, WP_Hook->apply_filters, bp_admin_init, do_action('bp_admin_init'), WP_Hook->do_action, WP_Hook->apply_filters, bp_core_admin_components_activation_handler, bp_core_install, bp_core_install_extended_profiles, dbDelta
[21-Jul-2021 15:40:39 UTC] WordPress database error Multiple primary key defined for query ALTER TABLE wp_bp_xprofile_data CHANGE COLUMN `id` id bigint(20) unsigned NOT NULL AUTO_INCREMENT PRIMARY KEY made by do_action('admin_init'), WP_Hook->do_action, WP_Hook->apply_filters, bp_admin_init, do_action('bp_admin_init'), WP_Hook->do_action, WP_Hook->apply_filters, bp_core_admin_components_activation_handler, bp_core_install, bp_core_install_extended_profiles, dbDelta
[21-Jul-2021 15:40:39 UTC] WordPress database error Multiple primary key defined for query ALTER TABLE wp_bp_xprofile_meta CHANGE COLUMN `id` id bigint(20) NOT NULL AUTO_INCREMENT PRIMARY KEY made by do_action('admin_init'), WP_Hook->do_action, WP_Hook->apply_filters, bp_admin_init, do_action('bp_admin_init'), WP_Hook->do_action, WP_Hook->apply_filters, bp_core_admin_components_activation_handler, bp_core_install, bp_core_install_extended_profiles, dbDelta
```

**Comment by ExoGeek** [comment]
Related : https://github.com/buddyboss/buddyboss-platform/issues/2628

**Comment by marienel21** [comment]
Hi @ianmjones @ExoGeek, 

The issue mentioned above is released in BuddyBoss Platform v1.7.6

You can check release notes here:
https://www.buddyboss.com/resources/buddyboss-platform-releases/1-7-6/

Please update the plugin to see the latest changes.

Thank you.
